### PR TITLE
SC-92 fixed image to fit in mobile and tablet for image carousel

### PR DESCRIPTION
**Pull request by kairen2000**
## Describe your changes (point form)

- Fixed image in mobile and tablet for cafe page image carousel
- From my memory, we care about mobile but thought it would be nice to make the image look nice in tablet as well, can always remove if we dont want it

## Jira Ticket 

[SC-92](https://webdevgang.atlassian.net/browse/SC-92)

## Include any Screenshots/Videos if relevant 

desktop
image

tablet
image

mobile
image

## Any other reminders reviewers should know? (`npm i`)

When testing, if you use responsive mode to change the media queries, the carousel gets a little weird

To fix this either:
1. switch the photo to resize the images
2. or refresh the page

[SC-92]: https://webdevgang.atlassian.net/browse/SC-92?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spire-coffee/src/component/carousel/ImageCarousel.tsx [hunk 1]
```diff
@@ -29,9 +29,10 @@ const ImageCarousel = () => {
...
         indicatorContainerProps={{
           style: {
-            marginTop: "1rem",
-            marginBottom: "1rem",
-            position: "relative",
+            position: "absolute",
+            bottom: "10px",
+            left: "50%",
+            transform: "translateX(-50%)",
             zIndex: 1,
```
Comment (kairen2000) [inline spire-coffee/src/component/carousel/ImageCarousel.tsx:36]: This is to keep the dots on the image like how it looks in dev. Without this the dots go below the image and the image does not take up the same amount of space like before

File: spire-coffee/src/component/carousel/ImageCarousel.tsx [hunk 2]
```diff
@@ -46,11 +47,25 @@ const ImageCarousel = () => {
...
-              height: "393px",
-              // objectFit: "cover",
+              height: {
+                xs: "275px",
+                sm: "550px",
+                md: "600px",
+                lg: "445px",
+              },
+              position: "relative", 
+              overflow: "hidden",
```
Comment (kairen2000) [inline spire-coffee/src/component/carousel/ImageCarousel.tsx:57]: This is what was making the images static before, so to fix it just needed to adjust the height depending on screen size, this is how tablet was added so if we dont want tablet i can remove the md screen size

File: spire-coffee/src/component/carousel/ImageCarousel.tsx [hunk 3]
```diff
@@ -46,11 +47,25 @@ const ImageCarousel = () => {
...
+            <img
+              src={item.imagePath}
+              alt={item.imagePath}
+              style={{
+                width: "100%",
+                height: "100%",
+                objectFit: "cover",
+              }}
+            />
           </Box>
```
Comment (kairen2000) [inline spire-coffee/src/component/carousel/ImageCarousel.tsx:69]: this is so the image takes up all the space it has in the container its in

**Review by bettyanhn — APPROVED** [review]
nice work T_T them dots

**Pull request merged**